examples/userapi/07_sparse_operations.ipynb

```diff
@@ -0,0 +1,889 @@
+{
```

Comment:
<p>Not sure how helpful this figure is.</p><p>Could be better probably with less points, to see understand weights better</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/2252/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>